engine/src/mac-av-player.mm

```diff
@@ -162,7 +160,8 @@ static CVReturn MyDisplayLinkCallback (CVDisplayLinkRef displayLink,
     bool m_finished : 1;
     bool m_has_invalid_filename : 1;
     bool m_mirrored : 1;
-
+    
+    void HandleCurrentTimeChanged(void);
```

Comment:
Probably best to move this back to where it was to eliminate the unnecessary churn in the source.